.cloudbuild/integration.yaml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://json.schemastore.org/cloudbuild.json
```

Comment:
Any good names for this file that don't overlap with "continuous integration" so much?

Reply:
`integration_test.yaml`? Open to suggestions.

Reply:
That has the same overlap. e2e.yaml? core-test.yaml?